### ISSUE TITLE
yiisoft/yii2-codeception 

### ISSUE THREAD

**Issue by medeyacom** [samdark/yii2-shop#34]
yiisoft / yii2-codeception - is no longer relevant. It is registered in the dependencies samdark / yii2-shop

  I decided to try installing $ composer require dmstr / yii2-adminlte-asset "^ 2.1" in yi2-shop (samdark) and connect nested-sets and now the composer got out: 'Package yiisoft / yii2-codeception is abandoned, you should avoid using it. Use codeception / codeception instead '. in composer.json - "yiisoft / yii2-codeception": "*",

**Comment by samdark** [comment]
That's correct. I can merge pull requests in but I don't plan to fix it myself.

**Comment by COKJLAH** [comment]
![image](https://user-images.githubusercontent.com/58906305/78493792-ecf8b500-7751-11ea-890e-38d3879647bf.png
Tell me how to solve these problems if necessary I can send the project
